common/include/pcl/experimental/executor/trait/can_require.hpp

```diff
@@ -0,0 +1,84 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2020-, Open Perception, Inc.
+ *  Author: Shrijit Singh <shrijitsingh99@gmail.com>
+ *
+ */
+
+#pragma once
+
+#include <pcl/experimental/executor/trait/common_traits.hpp>
+#include <type_traits>
+
+namespace executor {
+
+namespace detail {
+
+/**
+ * \brief Checks if the given Executor supports the Property
+ *
+ *\details This is checked through a template variable static_query_v which is
+ * provided by all properties
+ */
+template <typename Executor, typename Property>
+using contains_property =
+    std::is_same<std::remove_const_t<decltype(
+                     Property::template static_query<Executor>::value)>,
+                 Property>;
+}  // namespace detail
```

Comment:
Is `base_property.h` a better location for this?

Reply:
Yeah. Moved.